python/core/auto_generated/symbology/qgsmarkersymbollayer.sip.in

```diff
@@ -867,8 +867,9 @@ class QgsFontMarkerSymbolLayer : QgsMarkerSymbolLayer
 #include "qgsmarkersymbollayer.h"
 %End
   public:
+
     QgsFontMarkerSymbolLayer( const QString &fontFamily = DEFAULT_FONTMARKER_FONT,
-                              QChar chr = DEFAULT_FONTMARKER_CHR,
+                              QString string = DEFAULT_FONTMARKER_CHR,
```

Comment:
string -> "characters"?

Reply:
Actually, I think here I need to revert to chr, see API break comment below.